data/cloud-docs-nav-data.json

```diff
@@ -378,20 +383,49 @@
       { "title": "User Tokens", "path": "api-docs/user-tokens" },
       { "title": "Users", "path": "api-docs/users" },
       { "title": "Variables", "path": "api-docs/variables" },
-      { "title": "VCS Events", "path": "api-docs/vcs-events" },
-      { "title": "Workspaces", "path": "api-docs/workspaces" },
-      {
-        "title": "Workspace-Specific Variables",
-        "path": "api-docs/workspace-variables"
-      },
-      {
-        "title": "Workspace Resources",
-        "path": "api-docs/workspace-resources"
-      },
       {
         "title": "Variable Sets",
         "path": "api-docs/variable-sets"
       },
+      { "title": "VCS Events", "path": "api-docs/vcs-events" },
+      {
+        "title": "Workspaces",
+        "routes": [
+          { "title": "Overview", "path": "api-docs/workspaces" },
+          {
+            "title": "CRUD Actions",
+            "path": "api-docs/workspaces/workspace-crud-actions"
+          },
+          {
+            "title": "Lock Mechanism",
+            "path": "api-docs/workspaces/workspace-lock-mechanism"
+          },
+          {
+            "title": "SSH Key Assignment",
+            "path": "api-docs/workspaces/workspace-ssh-key-assignment"
+          },
+          {
+            "title": "Remote State Consumers",
+            "path": "api-docs/workspaces/workspace-remote-state-consumers"
+          },
+          {
+            "title": "Tags",
+            "path": "api-docs/workspaces/workspace-tags"
+          },
+          {
+            "title": "Workspace-Specific Variables",
+            "path": "api-docs/workspaces/workspace-variables"
+          },
+          {
+            "title": "Resources",
+            "path": "api-docs/workspaces/workspace-resources"
+          },
+          {
+            "title": "Appendix",
```

Comment:
I know "Related Resources" has been handled somewhat inconsistently throughout the API docs, and this is also the first time we've had to really handle it with separated out pages on the same topic like this. But I think the way we're doing it could be clearer. "Appendix" as the name for this page doesn't make it super clear what users will get when they come to this page (related resources). 

I had two thoughts - lmk what you think. 
- Can we rename this page to "Endpoint Related Resources" or something similar? I want folks to understand what they're getting when they click.
- I wonder if at the bottom of each page, we should add a section called "Related Resources" and then point folks toward the common page. This way, we're notifying folks that there ARE related resources, but not repeating all of the content on the common page. (example below)

## Related Resources

Workspace GET endpoints can optionally return related resources when requested with the `include` query parameter. Refer to [Endpoint Related Resources](link) for a full list.

Reply:
I actually think this is a really good idea TBH.  I had the same thought but didn't want to change the standardization on this.  I think this is absolutely reasonable and would make it easier for users to navigate throughout the _Workspaces_ stuff.  👍

Reply:
Oh, just of note -- I'll update this, but the only thing of note is that at least on the SideNav it might cause some confusion with plain ol' _Resources_ for workspaces so I think I'm going to make sure the title is "API Related Resources" for the SideNav (since it's nested), but on the page itself it will be "Workspaces API Related Resources" 👍